src/libraries/Microsoft.PowerFx.Core/IR/CoercionMatrix.cs

```diff
@@ -346,6 +346,10 @@ private static CoercionKind GetToStringCoercion(DType fromType)
             {
                 return CoercionKind.ViewToText;
             }
+            else if (DType.Guid.Accepts(fromType))
+            {
+                return CoercionKind.GuidToText;
+            }
```

Comment:
This is shared by PAClient - can it impact Canvas apps?

Reply:
See PR 5860239 in PA Client. Yes, there's a potential impact on canvas apps, if an author was using a GUID in a place that expected a string and didn't permit coercion. I don't think there's anything matching that description, but I could have overlooked something. This is a similar change to what we did with DateTime->Number coercion a few months back.